indigo/distribution.yaml

```diff
@@ -780,6 +780,14 @@ repositories:
       url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
       version: master
     status: maintained
+  bowpmap_ros-release:
+    release:
+      packages:
+      - bowpmap_ros
+      tags:
+        release: release/indigo/{package}/{version}
+      url: https://github.com/kejriwalnishant1990/bowpmap_ros.git
```

Comment:
I looked at this repository and it does not have most of the tags or branches for indigo. It should have tags and branches for all the target distros. It looks like non of the generators have run successfully or their results have not been pushed to the repository.